src/database/database.cpp

```diff
@@ -38,7 +38,8 @@ Database::Database()
     Poco::Data::SQLite::Connector::registerConnector();
 }
 
-aos::Error Database::Init(const std::string& dbPath, const std::string& migrationPath)
+aos::Error Database::Init(
```

Comment:
Can we pass config reference instead?

Reply:
There is no entry in the config for dbPath:
App class uses Poco::Path(config.mValue.mWorkingDir, cDBFileName).toString(),